app/validators/profile_validator.rb

```diff
@@ -16,7 +16,10 @@ def validate(record)
     # NOTE: The summary is a base profile field, which we add to all new Forem
     # instances, so it should be safe to validate. The method itself also guards
     # against the field's absence.
-    record.errors.add(:summary, I18n.t("validators.profile_validator.too_long")) if summary_too_long?(record)
+    if summary_too_long?(record)
+      record.errors.add(:summary, I18n.t("validators.profile_validator.too_long"),
+                        message: "bio is too long")
```

Comment:
I could be wrong, but I think we still want the error message produced via the i18n mechanism. I believe [`errors.add`](https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add) wants the second argument to be a `type` of error (and is `:invalid` by default). So, the underlying problem here might be that we're setting `type` when we should be setting `message`. Possibly something like:

```suggestion
      record.errors.add(:summary,
                        message: I18n.t("validators.profile_validator.too_long"))
```

To get the error message to say `Bio` instead of `Summary`, you'll want to add attribute translation to the model (`extend ActiveModel::Translation`) and add the attribute translation (in `config/locales/models`):

```
  attributes:
    summary: Bio
```

Let me know if you want to pair on any of this!